torch/_inductor/comms.py

```diff
@@ -171,10 +214,37 @@ def exposed_communication_time(collective_snode, remaining_snodes):
                 # we can ignore it. Otherwise, it's the end of the road for overlap opportunities
                 break
 
-            compute_time += runtimes[snode]
+            if isinstance(snode, GroupedSchedulerNode) and snode.temp_grouping:
+
+                def _fn(_snode):
```

Comment:
nit, move this fn def to above next to def of compute_time int, and rename it to `def accumulate_time` then call it in both branches (temp_visit(accumulate)) vs accumulate(snode))